linter.py

```diff
@@ -56,83 +54,11 @@ class Flake8(PythonLinter):
         '--max-line-length=': None,
         '--max-complexity=': -1,
         '--jobs=': '1',
-        '--show-code=': False
+        'show-code': False,
```

Comment:
That's essentially the thing to fix #35. Double '--' means that this is an argument for flake8, but we want it to be an internal argument for the linter.